db_to_csv.py

```diff
@@ -13,21 +14,40 @@ def __init__(self, limit=10):
         self.limit = limit
 
     def copy(self):
-        sql = "\COPY (SELECT {columns} FROM {table} ORDER BY random() LIMIT {limit}) TO '{dest}' WITH (FORMAT TEXT, DELIMITER ',')".format(
-            table=self.table,
-            columns=','.join(self.columns),
-            dest=self.dest,
+
+        # Erase the contents of the file
+        open(self.dest, 'w').close()
+
+        LOWEST_UUID = "00000000-0000-0000-0000-000000000000"
+        KEEP = 3
+
+        for i in xrange(self.limit):
+            got_a_row = False
+            while not got_a_row:
+                high_uuid = str(uuid.uuid4())
+                low_uuid = high_uuid[:KEEP] + LOWEST_UUID[KEEP:]
+                sql = "\COPY (SELECT {columns} FROM {table} WHERE id BETWEEN '{low_uuid}' and '{high_uuid}' ORDER BY id DESC LIMIT 1) TO stdout WITH (FORMAT TEXT, DELIMITER ',')".format(
```

Comment:
I couldn't figure out how to calculate the `low_uuid` within the query (mostly because I couldn't figure out how to reuse the output of a function in a single query), but if you have a suggestion about how to do that, I would love to see it.